code/modules/antagonists/wizard/equipment/enchanted_clown_suit.dm

```diff
@@ -0,0 +1,104 @@
+/// A spell which gives you a clown item
+/datum/action/cooldown/spell/clown_pockets
+	name = "Acquire Clowning Implement"
+	desc = "Pull an item out of your mysteriously expansive pants."
+	button_icon = 'icons/obj/clothing/masks.dmi'
+	button_icon_state = "clown"
+	school = SCHOOL_CONJURATION
+	spell_requirements = NONE
+	cooldown_time = 30 SECONDS
+	cooldown_reduction_per_rank = 2 SECONDS
+	/// Amount of time it takes you to rummage around in there
+	var/cast_time = 3 SECONDS
+	/// Set to true if you fail to pull an item out.
+	var/interrupted = FALSE
+	/// True while currently casting the spell
+	var/casting = FALSE
+	/// List of prank implements you can find in your pockets
+	var/static/list/clown_items = list(
+		/obj/item/grown/bananapeel = 5,
+		/obj/item/food/pie/cream = 5,
+		/obj/item/bikehorn = 5,
```

Comment:
alphabetize this entire list. there's a few VSC extensions that do this for you rather quickly and pain-free